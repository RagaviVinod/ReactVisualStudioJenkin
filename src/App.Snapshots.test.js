import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

it("sets submit button label ",()=>{
 const tree = renderer.create(<App/>);
 expect(tree).toMatchSnapshot(); 
});


