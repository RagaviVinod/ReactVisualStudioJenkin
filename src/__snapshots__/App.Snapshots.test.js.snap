// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sets submit button label  1`] = `
<div>
  HelloWorld
</div>
`;
